spec/features/activity_page_spec.rb

```diff
@@ -62,7 +62,7 @@
       it 'filters the courses to my courses' do
         # Admin is admin of course 1, should only see user1's revision
         # when checked
-        click_link 'Recent Activity'
+        visit '/recent-activity'
```

Comment:
The headless webkit browser wasn't able to click the link due to the `position: fixed` header nav covering it up after some style changes I made. This was the easier fix, but I can switch to using selenium for this test instead if you'd rather.